crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -809,6 +809,18 @@ void LateUpdatePosition()
             // maintain y coordinate - sea level
             pos.y = Root.position.y;
 
+            // Don't land very close to regular positions where things are likely to snap to, because different tiles might
+            // land on either side of a snap boundary due to numerical error and snap to the wrong positions. Nudge away from
+            // common by using increments of 1/60 which have lots of factors.
```

Comment:
Should we add `:OceanGridPrecisionErrors` to keep track?

Reply:
Done, thanks